mlflow/cli.py

```diff
@@ -132,6 +132,7 @@ def ui(file_store_path):
     server.app.run("0.0.0.0")
 
 
+cli.add_command(mlflow.tensorflow.commands)
```

Comment:
We should remove this since we aren't adding any TensorFlow-specific CLI commands (?)